pkg/ingester/transfer.go

```diff
@@ -214,6 +214,13 @@ func (i *Ingester) TransferTSDB(stream client.Ingester_TransferTSDBServer) error
 	fromIngesterID := ""
 
 	xfer := func() error {
+
+		dir, err := ioutil.TempDir("", "tsdb_xfer")
```

Comment:
I would suggest to rename `dir` to `tempDir` for clarity.

Reply:
done